mkdocs/docs/HPC/only/gent/2023/shinx.md

```diff
@@ -0,0 +1,113 @@
+and `victini`, which will be retired on **Monday 05 February 2024**.
+
+It is primarily for regular CPU compute use.
+
+This cluster consists of 48 workernodes, each with:
+
+* 2x 96-core AMD EPYC 9654 (Genoa @ 2.4 GHz) processor;
+* ~360 GiB of RAM memory;
+* 400GB local disk;
+* NDR-200 InfiniBand interconnect;
+* RHEL9 as operating system;
+
+To start using this cluster from a terminal session, first run:
+```
+module swap cluster/shinx
+```
+
+You can also start (interactive) sessions on `shinx` using the [HPC-UGent web portal](../../../web_portal.md).
+
+### Differences compared to `swalot` and `slaking`.
+
+#### CPUs
+
+The most important difference between `shinx` and `swalot`/`victini` workernodes is in the CPUs:
+while `swalot` and `vicitini` workernodes featured *Intel * CPUs, `shinx` workernodes have `AMD Genoa` CPUs.
+
+Although software that was built on a `swalot` or `vicitini` workernode with compiler options that enable architecture-specific
+optimizations (like GCC's `-march=native`, or Intel compiler's `-xHost`) might still run on
+a `shinx` workernode, it is recommended to recompile the software to benefit from the support for
+`AVX-512` vector instructions (which is missing on `swalot`).
+
+#### Cluster size
+
+The `shinx` cluster is significantly bigger than `swalot` and `victini` in number of cores, and numer of cores per workernode,
+but not in number of workernodes. In particular, requesting all cores via `ppn=all` might be something to reconsider.
+
+The amount of available memory per core is `1.9 GiB`, which is lower then the `swalot` ndoes which had `6.2 GiB` per core
+and the vicitini nodes that had `2.5 GiB` per core.
+
+
+### Comparison with `doduo`
+
+As `doduo` is the current largest CPU cluster of the UGent Tier-2 infrastructure, and it is also based on `AMD EPYC` CPUs,
+we would like to point out that, roughly speaking, one `shinx` node is equal to 2 `doduo` nodes.
+
+Although software that was built on a `doduo` workernode with compiler options that enable architecture-specific
+optimizations (like GCC's `-march=native`, or Intel compiler's `-xHost`) might still run on
+ a `shinx` workernode, it is recommended to recompile the software to benefit from the support for
+`AVX-512` vector instructions (which is missing from `doduo`).
+
+### Other remarks
+
+* Possible issues with `OpenMP` thread pinning: we have seen, especially on `Tier-1 dodrio` cluster, that in certain cases
+thread pinning occurs that also affects
+
+---
+
+## Shinx pilot phase (23/10/2023-20/05/2024)
+
+Because the delivery time of the infiniband network is very high, we only expect to have all material end of February 2024.
+However, all the workernodes will already be delivered in the week of 20 October 2023
+
+As such, we will have an extended pilot phase in 3 stages:
+
+### Stage 0: 23/10/2023-17/11/2023
+
+* Minimal cluster to test software and nodes
+    * Only 2 or 3 nodes available
+    * FDR or EDR infiniband network
+    * EL8 OS
```

Comment:
Why not EL9 immediately?

Reply:
we are far from ready for el9, and the hardware was delivered extremely quick, so we are taken by surprise